perun-core/src/main/java/cz/metacentrum/perun/core/impl/ExtSourcesManagerImpl.java

```diff
@@ -138,10 +139,9 @@ public ExtSource createExtSource(PerunSession sess, ExtSource extSource) throws
 			es.setId(extSource.getId());
 			es.setName(extSource.getName());
 			es.setType(extSource.getType());
-			es.setAttributes(extSource.getAttributes());
 
 			// Now store the attributes
-			Map<String, String> attributes = extSource.getAttributes();
+			Map<String, String> attributes = getAttributes(extSource);
```

Comment:
This  is not correct. These attributes do not exists before creating extSource. We need to save them separately. For example in method createExtSource use Map<String,String> attributes like argument. And then save them to DB there.